thirtyinch/src/main/java/net/grandcentrix/thirtyinch/serialize/TiPresenterSerializer.java

```diff
@@ -0,0 +1,40 @@
+package net.grandcentrix.thirtyinch.serialize;
+
+import android.support.annotation.NonNull;
+
+import net.grandcentrix.thirtyinch.TiPresenter;
+import net.grandcentrix.thirtyinch.TiView;
+
+/**
+ * Interface to serialize and deserialize presenters. The interface should be implemented, if
+ * a {@link TiPresenter} should be restored, after a process has died, but the UI component is
+ * being recreated.
+ */
+public interface TiPresenterSerializer {
```

Comment:
I think both changes are safe, because nothing of that was exposed to the public API, yet. Do you want me to create a different PR?